# This is a message to hold data for the ECU (electronic control unit)
#
# angles should be in degrees

float32 throttle        # throttle
float32 str_ang         # steering angle [rad]
